src/overlays/actors/ovl_Obj_Tokei_Tobira/z_obj_tokei_tobira.c

```diff
@@ -36,16 +37,132 @@ static InitChainEntry D_80ABD750[] = {
     ICHAIN_VEC3F_DIV1000(scale, 100, ICHAIN_STOP),
 };
```

Comment:
You can't see it here, but the `D_80ABD750` is an InitChain. We rename all the InitChains as `sInitChain` (as the comment above it suggest it). Please rename it